internal/services/appservice/migration/linux_web_app.go

```diff
@@ -1249,7 +1244,7 @@ func (l LinuxWebAppV0toV1) Schema() map[string]*pluginsdk.Schema {
 											},
 
 											"status_code": {
-												Type:     pluginsdk.TypeList,
+												Type:     pluginsdk.TypeSet,
```

Comment:
This needs to be put back to `TypeList` as it represents the schema at the point in time of the migration, where it was a list.